plugins/field-grid-dropdown/README.md

```diff
@@ -0,0 +1,60 @@
+# blockly-field-grid-dropdown [![Built on Blockly](https://tinyurl.com/built-on-blockly)](https://github.com/google/blockly)
+
+A [Blockly](https://www.npmjs.com/package/blockly) dropdown field with grid layout.
+
+![](https://github.com/google/blockly-samples/raw/master/plugins/field-grid-dropdown/readme-media/dropdown.png)
+
+## Installation
+
+### Yarn
+```
+yarn add blockly-field-grid-dropdown
+```
+
+### npm
+```
+npm install blockly-field-grid-dropdown --save
+```
+
+## Usage
+
+### JavaScript
+```js
+import * as Blockly from 'blockly';
+import {FieldGridDropdown} from 'blockly-field-grid-dropdown'; import {FieldGridDropdown} from './index';
```

Comment:
This is importing FieldGridDropdown twice, from different places, on the same line.